src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -224,3 +224,19 @@ hooks.addPureHook('PATCH', 'resin', 'device', {
 		}
 	},
 });
+
+hooks.addPureHook('PATCH', 'resin', 'device', {
```

Comment:
We might also need to change the device move implementation as well (earlier in this file), since moving a device atm deletes all service installs & creates the ones of the new app.
See: https://github.com/balena-io/open-balena-api/blob/c40079c5fa6cef1b5744493b96e7f2fcf3ccc594/src/features/ci-cd/hooks/service-installs.ts#L164-L174

In order to not lose the services installs created by this hook, we probably need to avoid deleting supervisor release related service installs, right?

Reply:
Yeah, I had thought about that and was going to do it in another PR.  @Page- What is the best way to do this?